theme/styles/abstract/accessible.sass

```diff
@@ -1,6 +0,0 @@
-%accessible
```

Comment:
důvod odstranění této šikovné abstraktní třídy?

Reply:
Daleko líp to řeší `aria` atributy, které jsou na to stavěné, a nejsou takový hack.

Reply:
Tady se právě bojím toho, že aria atributy jsem ještě od kodérů při code review neviděl a když schovávají obsahové/navigační prvky, dělají to natvrdo přes `display:none`. A to je pak zlo největší.

Reply:
Já jsem při code review neviděl nikoho používat `accessible`.

`aria` má dokumentaci a není to hack s `!important` a dvojitým `clip: rect`. IMHO je to udržitelnější řešení, které je snazší ostatní kodéry neučit.

Reply:
`clip: rect` není hack, ale používaná technika pro přístupné schování obsahu, vznikla dávno před existencí WIA-ARIA. Stejnou techniku používá Bootstrap pro třídu `.sr-only`:
```css
.sr-only{
  position:absolute;
  width:1px;
  height:1px;
  padding:0;
  overflow:hidden;
  clip:rect(0,0,0,0);
  white-space:nowrap;
  border:0
}
```

No ale takže máme obecně problém s přístupností webů. O to větší důraz na možnosti, jak jít tomu naproti. Osvěta je jedna věc (a kdo ji bude dělat?), pomocné nástroje druhá, stejně důležitá.

Pokud chceme toto odstranit, prosím o náhradu.

Reply:
Proč je rect/accessible hack:

1. Deprecated
2. Název property + value nevystihuje záměr.
3. V `accessible` se nezvykle vyskytuje dvakrát za sebou. 
4. Vyžaduje zbytečný node v DOMu navíc.
5. Používá `!important`, kterému se všude snažíme vyhnout a používáme ho jen v případech, kdy něco hackujeme, třeba framework.
6. Zneužívá styly pro dosažení cíle, ke kterému jsou určeny jiné nástroje (`aria`).

Za předpokladu, že `accessible` používáš jen ty, jako náhrada bude stačit, že o `aria` výš.

Osvěta u ostatních by jistě nebyla na škodu. Ale tu osvětu by si dříve zasloužilo i `accessible`. Alespoň komentářem v kódu.

Pokud díky `accessible` jsou naše weby třeba jen o trochu víc dostupné, nechme to kódu. (Já to nepoužívám.)

Reply:
1. in favor of `clip-path`, který ale není podporovaný v IE např. vůbec (ani Edge)
2. accessible = able to be reached or entered, takže pojmenování mi přijde odpovídající
3. první je starší definice, proto ten komentář `/* IE6, IE7 */`, druhá standardní
4. to bude vyžadovat i řešení pomocí aria atributů
5. nebo vynucujeme. Tady absolutní pozice, aby byl prvek vždy vyjat z DOMu
6. i s aria atributy budeš pomocí CSS skrývat věci, které nechceš, aby ti pronikaly do designu. Zároveň nevím, že by se aria atributy indexovaly, takže třeba strukturu dokumentu (nadpisy) bys rozhodně měl držet v html elementech.

Celá pointa této třídy byla, že je to ověřená (ok, ale stará) technika, která funguje a nemusíš jí znova vymýšlet/googlit.

Reply:
Za mě ji klidně nechme. Třeba revertnutím commitu.